proto/cosmos/protocolpool/v1/tx.proto

```diff
@@ -101,3 +107,54 @@ message MsgClaimBudgetResponse {
       [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
   ;
 }
+
+// MsgAddContinuousFunds defines a message for adding continuous funds.
+message MsgAddContinuousFunds {
+  option (cosmos.msg.v1.signer)      = "authority";
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  // title is the title of the funds.
+  string title = 1;
+  // description of the funds.
+  string description = 2;
+  // authority is the address that controls the module (defaults to x/gov unless overwritten).
+  string authority = 3 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // recipient address of the account receiving funds.
+  string recipient = 4 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // metadata is any arbitrary metadata attached.
+  string metadata = 5;
+  // percentage is the percentage of funds to be allocated from Community pool share on block by block.
+  string percentage = 6 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
+    (gogoproto.nullable)   = false
+  ];
+  repeated cosmos.base.v1beta1.Coin cap = 7
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+  // Optional, if expiry is set, removes the state object when expired.
+  google.protobuf.Timestamp expiry = 8 [(gogoproto.stdtime) = true];
+}
+
+// MsgAddContinuousFundsResponse defines the response to executing a
+// MsgAddContinuousFunds message.
+message MsgAddContinuousFundsResponse {}
+
+// MsgCancelContinuousFundProposal
+message MsgCancelContinuousFundProposal {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the account address of authority.
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // recipient_address is the account address of recipient whose funds are to be cancelled.
+  string recipient_address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
So we cannot have two funds streaming for the same address at a time. Is it expected? If not let's have a unique key for each fund so we can just remove fund by fundID or something.

Reply:
this is expected, otherwise we would need to add a override method, this way its simpler. New addresses are cheap and with the accounts module many addresses can be owned by a single signer